android/gradle.properties

```diff
@@ -26,4 +26,4 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.75.1
\ No newline at end of file
+FLIPPER_VERSION=0.93.0
```

Comment:
Were you able to open a pool with flipper version 0.93.0?

I recently updated flipper and the Android app started crashing unexpectedly again. That was on version 0.98.0, so not sure if it also affects this version.

https://github.com/hyperledger/indy-sdk/issues/2346#issuecomment-890172894